lib/api/services/wallet_service.dart

```diff
@@ -1,210 +1,149 @@
+import 'dart:convert';
+import 'dart:typed_data';
+
+import 'package:arbor/api/responses/record_response.dart';
+import 'package:arbor/api/responses/records_response.dart';
 import 'package:arbor/api/responses/transaction_response.dart';
-import 'package:arbor/api/responses/wallet_address_response.dart';
+import 'package:arbor/bls/ec.dart';
+import 'package:arbor/bls/private_key.dart';
+import 'package:arbor/bls/schemes.dart';
+import 'package:arbor/clvm/program.dart';
+import 'package:arbor/core/utils/puzzles.dart';
 import 'package:arbor/models/transaction.dart';
-
-import '/models/models.dart';
+import 'package:bech32m/bech32m.dart';
+import 'package:bip39/bip39.dart';
+import 'package:hex/hex.dart';
 import 'package:http/http.dart' as http;
-import 'dart:convert';
 
+import '/models/models.dart';
 import '../responses.dart';
 import 'api_service.dart';
 
 class WalletService extends ApiService {
-  WalletService({this.baseURL = ApiService.baseURL});
+  @override
+  String baseURL = 'https://dev.arborwallet.com/api';
```

Comment:
@Rigidity , 'https://dev.arborwallet.com/api' suggests that there's a development server for arbor wallet. Correct?

Reply:
Yes, this will be reverted to https://arborwallet.com/api when the branch is merged. If there's a better way to set this, e.g. as an environment variable, .env file, or something else, let me know.